src/main/java/com/com/mock/preparation/ducat/Oops/arrayassignment/HCFArray.java

```diff
@@ -0,0 +1,22 @@
+package com.mock.preparation.ducat.Oops.arrayassignment;
+
+import java.util.Scanner;
+
+public class HCFArray {
+    public static void main(String[] args) {
+        int a, b, hcf = 0;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("enter the interger value: ");
```

Comment:
what is difference in line 9 and 11 ? both have same message. How input a and b are different?